Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateTimeExtractor.cs

```diff
@@ -216,5 +217,43 @@ public List<Token> SpecialTimeOfDate(string text)
             }
             return ret;
         }
+
+        // process case like "two minutes ago" "three hours later"
+        private List<Token> DurationWithBeforeAndAfter(string text)

```

Comment:
The same code in Microsoft.Recognizers.Text.DateTime/Extractors/BaseDateExtractor.cs. Can we only have one version?